packages/app/pages/AllProjectsPage/AllProjectsPage.tsx

```diff
@@ -119,7 +115,6 @@ class ProjectCard extends React.Component {
           <div {...this.currentTheme((this.i * 7) + 15, "item_row")}>
             <span {...this.currentTheme((this.i * 7) + 16, "row_image")}><img src={this.props.org.image} /></span>
             <Link href={`projects/${this.props.org.id}`} {...this.currentTheme((this.i * 7) + 17, "row_title")}>{this.props.org.name}</Link>
```

Comment:
This line exceeds the maximum line length

Reply:
Will run the linter and make changes